api/add_dimension.md

```diff
@@ -119,7 +92,7 @@ partitioning on column `time`, then add an additional partition key on
 
 ```sql
 SELECT create_hypertable('conditions', by_range('time'));
-SELECT add_dimension('conditions', by_hash('location', 4));
+SELECT add_dimension('conditions', by_range('time'), by_hash('location', 4));
```

Comment:
I don't think this change is correct. Looks like you are confusing `create_hypertable` and `add_dimension`.

`create_hypertable` requires adding at list a range dimension as the "primary" partitioning dimension. However, `add_dimension` is called _after_ `create_hypertable` to add additional dimensions, typically a space/hash dimension after the hypertable exists  (including its primary range/time dimension).

Reply:
Updated.